corokafka/impl/corokafka_producer_manager_impl.cpp

```diff
@@ -623,5 +606,25 @@ ProducerManagerImpl::findProducer(const std::string& topic) const
     return it;
 }
 
+int32_t ProducerManagerImpl::mapPartitionToQueue(const uint8_t* obj,
```

Comment:
I suggest renaming this to mapKeyToQueue() or mapKeyToThread() or similar; it's not mapping a partition.